### ISSUE TITLE
Gradle Plugin requires Kotlin Plugin to be 

### ISSUE THREAD

**Issue by knightzmc** [knightzmc/pdm#1]
For some bizarre reason, the Gradle Plugin seems to _sometimes_ need the Kotlin JVM plugin to be declared to function.

In this example 
```gradle
plugins {
    id 'java'
    id 'me.bristermitten.pdm' version '0.0.1'
}
```

Project loading in IntelliJ (and normal building) fails, claiming that `com/google/gson/Gson` is not found.

According to the published plugin, this should be resolved properly as Gson is explicitly declared as a dependency. Apparently, however it is not.

Adding the Kotlin JVM Plugin (`id 'org.jetbrains.kotlin.jvm' version '1.3.72'`) seems to resolve this.

The strange part is that the Kotlin Plugin doesn't provide Gson to my knowledge, so I have no idea what would be causing this - my best guess is that something is being published incorrectly.

**Comment by knightzmc** [comment]
To clarify, this is the specific error that occurs without having the Kotlin Plugin:

```
java.lang.NoClassDefFoundError: com/google/gson/Gson
	at me.bristermitten.pdm.PDMGenDependenciesTask.<init>(PDMGenDependenciesTask.kt:23)
	at me.bristermitten.pdm.PDM.apply(PDM.kt:45)
	at me.bristermitten.pdm.PDM.apply(PDM.kt:8)
```

**Comment by knightzmc** [comment]
This _should_ be fixed in the most recent version. However, since we can't deploy to the Gradle Plugins Repository without bumping the version (which should be avoided in the current state), there's no definitive way of testing yet.

**Comment by knightzmc** [comment]
Fix in the latest version (0.0.8)
